### PR TITLE
update models to support partitioned models

### PR DESCRIPTION
**Pull request by yunhaoling**
Issue: https://github.com/Azure/azure-sdk-for-python/issues/4247

1. Update models to support partitioning
2. add timezone info to enqueue_time, schedule_enqueue_time property in tests and message.py
3. fix bugs mentioned in the issue
4. add new test for partitioned models
5. new recording files generated by the legacy test

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

**Comment by annatisch** [comment]
Hey @yunhaoling - I'm going to temporarily close this PR for now, as we have to restructure the SDKs in the repo - and it looks like servicebus has been lowered in priority compared to event hubs etc.

Once the repo restructure is complete we can re-open the PR!

File: azure-servicebus/azure/servicebus/control_client/servicebusservice.py [hunk 1]
```diff
@@ -1111,7 +1111,8 @@ def create_event_hub(self, hub_name, hub=None, fail_on_exist=False):
         request = HTTPRequest()
         request.method = 'PUT'
         request.host = self._get_host()
-        request.path = '/' + _str(hub_name) + '?api-version=2014-01'
+        #request.path = '/' + _str(hub_name) + '?api-version=2014-01'
```
Comment (KieranBrantnerMagee) [inline azure-servicebus/azure/servicebus/control_client/servicebusservice.py]: Is there a reason to leave these comments in if we've moved to the x_ms_version above?

File: azure-servicebus/tests/test_partitioned_sessions.py [hunk 2]
```diff
@@ -726,7 +727,7 @@ def test_qsession_by_servicebus_client_list_sessions_with_receiver(live_serviceb
 
 @pytest.mark.liveTest
 def test_qsession_by_servicebus_client_list_sessions_with_client(live_servicebus_config, partitioned_session_queue):
-
+    pytest.skip("TODO: Investigation with the service team on the list_sessions api")
```
Comment (KieranBrantnerMagee) [inline azure-servicebus/tests/test_partitioned_sessions.py]: Is there an issue/backlog item tracking this?

File: azure-servicebus/azure/servicebus/control_client/_serialization.py [hunk 3]
```diff
@@ -228,18 +238,33 @@ def _convert_etree_element_to_queue(entry_element):
...
+            _convert_etree_element_to_authorization_rules_in_entity(queue, rules_nodes)
+
+
     if invalid_queue:
         raise AzureServiceBusResourceNotFound(_ERROR_QUEUE_NOT_FOUND)
 
     # extract id, updated and name value from feed entry and set them of queue.
     for name, value in _ETreeXmlToObject.get_entry_properties_from_element(
             entry_element, True).items():
+        if name == 'name' and '?' in str(value):
```
Comment (KieranBrantnerMagee) [inline azure-servicebus/azure/servicebus/control_client/_serialization.py]: Can we be more precise via endswith or something to that effect?

**Pull request closed without merging**